target/linux/ipq40xx/files-4.19/arch/arm/boot/dts/qcom-ipq4018-meshpoint-one.dts

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/* Copyright (c) 2019, CRISIS INNOVATION LAB d.o.o.
+ * Author: Robert Marko <robert@meshpoint.me>
+ */
+
+#include "qcom-ipq4018-jalapeno.dtsi"
+
+/ {
+	model = "Crisis Innovation Lab MeshPoint.One";
+	compatible = "cilab,meshpoint-one";
+
+	aliases {
+		led-boot = &led_status;
+		led-failsafe = &led_status;
+		led-running = &led_status;
+		led-upgrade = &led_status;
+	};
+
+	soc {
```

Comment:
From my experience with other targets, I'd have added a label to soc in the dtsi and used this label here:
&soc {
   i2c-gpio {
       ...
   };
};

I know that both versions work, so this more a comment than a suggestion. Maybe this is also treated differently in ipq40xx ...

Reply:
Well, since upstream and all other DTS code for ipq40xx behaves like this I don't see an advantage in applying another patch to the upstream DTSI and then referencing that node.
A while ago most nodes got labels but soc remained this way.

Reply:
Fine.